google_shopping.js

```diff
@@ -1,5 +1,85 @@
+//1.) The kind of results you're are dealing are shopping#products. 
+//Go through the items and find all results that have kind of shopping#product. 
+//How many are there? Where else is this count information 
+//stored in the search results?
+
 var data = require("./products.json")
+var newArray = [];
+
+for (var i = 0; i < data["items"].length; i++) {
+	if (data["items"][i]["kind"] === "shopping#product") {
+		newArray.push(data["items"][i]["id"]);
+	}
+}
+
+console.log(newArray.length);
+
+//The code above displays the number of shopping#products in the list.
+//The code below also displays this information.
+
+console.log(data["currentItemCount"]);
+
+console.log("THE FOLLOWING ITEMS ARE ON BACKORDER.")
+
+for (var i = 0; i < data["items"].length; i++) {
+	if (data["items"][i]["product"]["inventories"][0]["availability"] === "backorder") {
+		console.log(data["items"][i]["product"]["title"]);
+		console.log(" ");
```

Comment:
Instead of console.log(' ') to get a new line, you can also add + \n to your previous console log string to force a new line.